modules/ExpensifyNitroUtils/src/specs/ContactsModule.nitro.ts

```diff
@@ -0,0 +1,20 @@
+import type {HybridObject} from 'react-native-nitro-modules';
+
+type StringHolder = {
+    value: string;
+};
+
+export type Contact = {
+    firstName?: string;
+    lastName?: string;
+    middleName?: string;
+    phoneNumbers?: StringHolder[];
+    emailAddresses?: StringHolder[];
+    imageData?: string;
+    thumbnailImageData?: string;
+};
+export type ContactFields = 'FIRST_NAME' | 'LAST_NAME' | 'MIDDLE_NAME' | 'PHONE_NUMBERS' | 'EMAIL_ADDRESSES' | 'IMAGE_DATA' | 'THUMBNAIL_IMAGE_DATA' | 'GIVEN_NAME_KEY';
+
+export interface ContactsModule extends HybridObject<{ios: 'swift'; android: 'kotlin'}> {
+    getAll(keys: ContactFields[]): Promise<Contact[]>;
```

Comment:
I see this is only used once, so you could simplify things by just hard-coding the fields we query to `[FIRST_NAME, LAST_NAME, PHONE_NUMBERS, EMAIL_ADDRESSES, IMAGE_DATA]` since those are the only fields we deep to be using.

Reply:
Do you mean hardcoding fields on the native side? If yes, I don't see much benefit since the implementation already fetches the same amount of fields anyway. If you're referring to trimming the type definition in TS to only include fields we actually use, that makes more sense to me

Reply:
> If you're referring to trimming the type definition in TS to only include fields we actually use, that makes more sense to me

yeah, that's what I was thinking

Reply:
Got it, I'll trim it